pkg/cmd/cli/cmd/set/volume.go

```diff
@@ -532,6 +549,8 @@ func (v *VolumeOptions) getVolumeUpdatePatches(infos []*resource.Info, singular
 }
 
 func setVolumeSourceByType(kv *kapi.Volume, opts *AddVolumeOptions) error {
+	defaultMode, _ := strconv.ParseInt(opts.DefaultMode, 8, 32)
```

Comment:
I know it's not expected to happen, but I'd return the error anyway (for unit tests, etc).